src/ansys/hps/client/jms/api/project_api.py

```diff
@@ -657,17 +674,44 @@ def _delete_objects(self, objects: List[Object], obj_type: Type[Object]):
 
 def _download_files(project_api: ProjectApi, files: List[File]):
     """
-    Download files directly using the fs REST gateway.
+    Download files directly using data transfer worker.
 
-    This is a temporary implementation for downloading files. It is to be
-    replaced with direct ansft calls, when it is available as a Python package.
     """
 
+    project_api.client._start_dt_worker()
+    out_path = os.path.join(os.path.dirname(__file__), "downloads")
```

Comment:
Can we use a proper temp dir instead? https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory

Reply:
updated code